projects/angular/src/emphasis/_labels.clarity.scss

```
@@ -157,6 +185,34 @@
     }
 
     &.clickable {
+      @include mixins.css-var(
+        font-size,
+        clr-label-clickable-font-size,
+        label-variables.$clr-label-font-size,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        font-weight,
+        clr-label-clickable-font-weight,
+        label-variables.$clr-label-font-weight,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        letter-spacing,
+        clr-label-clickable-letter-spacing,
+        label-variables.$clr-label-letter-spacing,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        line-height,
+        clr-label-clickable-line-height,
+        // line-height needed to be adjusted to force center alignment
+        label-variables.$clr-label-font-size + variables.$clr_baselineRem_1px,
+        variables.$clr-use-custom-properties
+      );
+    }
+
+    &.clickable:not(.disabled) {
       &:hover,
       &:active {
```

Active have a separate handler.
```suggestion
      &:hover {
```